link/src/main/webapp/app-root/clientlibs/authoring/dialog/js/validation.js

```diff
@@ -48,4 +48,25 @@
     }
   });
 
+  $(document).on("foundation-contentloaded", function(e) {
```

Comment:
I think this should be put in a separate js file, because the logic has nothing todo with validation.

Reply:
Done